cmd/kogito/command/install/kogito_operator.go

```diff
@@ -76,8 +78,14 @@ func (i *installKogitoOperatorCommand) Exec(cmd *cobra.Command, args []string) e
 		return err
 	}
 
-	if _, err = shared.MustInstallOperatorIfNotExists(i.flags.Namespace, i.flags.Image, i.Client, false); err != nil {
-		return err
+	if _, err = shared.ForceInstallOperator(i.flags.Namespace, i.flags.Image, i.flags.Force, i.Client); err != nil {
+		return nil
+	}
+
+	if !i.flags.Force {
```

Comment:
Why not refactor to `InstallOperatorIfNotExists` and pass the flag to it? This way we can encapsulate the install algorithm there.

Reply:
+1

Reply:
@ricardozanini do you say "MustInstallOperatorIfNotExists"? It is used in many other places in the code. This change would impact several places in the project, which is unnecessary as the only place that needs to "force" is the installation. Makes sense? thanks

Reply:
I don't think we need a specific behavior and specific function. 
The `force` is specific to the CLI execution. I would just keep it in the exec command if possible:
```
if i.flags.Force {
    // InstallOperatorWithYamlFiles <= We would need to make it public
} else {
    // MustInstallOperatorIfNotExists
}
```
But could also make sense to update `MustInstallOperatorIfNotExists` now with a new boolean if we plan to use this `force` in another place ?

Reply:
Please don’t expose the Install...YamlFiles. This function is intended to be private and encapsulate a way to install the operator. The delegate function (Must..) is responsible to call it. I will wrote another function to deal with subscriptions later that will make use of it. 

The purpose is to create another delegate function that just installs the operator whether or not it’s installed in the cluster. 

And @msmagnanijr please remove the alias “f” of the flag since I will use it to “follow” output logs in future releases.